### ISSUE TITLE
Display what form the user's wrong interpretation would have resulted in

### ISSUE THREAD

**Issue by hftf** [camilstaps/HebrewParseTrainer#17]
A comparison of two Hebrew forms helps understand where the user went wrong in parsing.

**Comment by camilstaps** [comment]
I agree, and would be happy to look at a merge request that would add this functionality. However, since the app does not have any knowledge of Hebrew grammar, it would not be able to generate the forms itself; it would only be able to look up whether the form the user entered exists in the database. Thus, if the user enters a form that does not exist in the database, it would not be possible to provide a comparison.
